websites/recipe-website/export/package.json

```diff
@@ -8,6 +8,7 @@
     "start:test": "CONTENT_DIRECTORY=test-content next start",
     "build": "next build",
     "start": "serve out -l 3001",
+    "deploy": "netlify deploy -d websites/recipe-website/export/out",
```

Comment:
these commands seem to be run relative to the monorepo root without an option to do so in the child package. might benefit from a rework with more direct calls.